src/Uno.UI/UI/Xaml/Media/LinearGradiientBrush.faux.cs

```diff
@@ -0,0 +1,120 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using Uno.UI.Xaml.Media;
+using Windows.Foundation.Collections;
+using Windows.UI;
+
+namespace Microsoft.UI.Xaml.Media;
+
+public partial class LinearGradientBrush
+{
+	private Color? _majorStopColor;
+	private SolidColorBrush? _fauxOverlayBrush;
+
+	/// <summary>
+	/// Gets or sets a flag indicating whether this brush should use
+	/// major gradient solid brush color in case the target platform
+	/// cannot apply the linear gradient to border correctly.
+	/// </summary>
+	public bool SupportsFauxBorder
```

Comment:
Can we have a doc page on that topic?

Reply:
Ha! Actually not, got rid of it 😂😂 . Instead we just specify the `FallbackColor` for the gradient, which will be applied when the border linear gradient is not possible to display correctly on the given target. As the faux presenter is only applied in our code currently, the new simplified logic now just picks the "other" color from the two gradient stops. The presenter is really for the specific use case of the Elevation border brushes in fluent styles, and I don't expect people to use it externally, but will add some basic docs on what it does anyway